### DISCUSSION TITLE
Does the system inside qemu need to access the Internet?

### DISCUSSION THREAD

**Discussion by click0** [depenguin-me/depenguin-run#56] [Ideas]
Does the system inside qemu need to access the Internet?
Or do we have all ISOs (will) be designed for offline installation?

**Comment by bretton** [comment]
yes, freebsd install files and packages are installed at different stages, and ssh keys retrieved from URLs, depending on which approach used. 

zfsinstall might not require internet access.

simply bundling a few packages into the image raised the size a lot, bundling installer ISOs might be problematic. 

maybe run.sh could have an option to include an ISO as third mounted-in option (incl download), but for now we're assuming remote access to rescue console & qemu is possible.
